src/style/color-palette-primary.scss.bak

```diff
@@ -104,6 +104,7 @@
     --color-glaucous-light: 135, 174, 193; // #87aec1
```

Comment:
Issues when running in lime-webclient `main`:

There are some shadows that are markedly more pronounced when the system is in dark mode. That's no show-stopper though.

Worse is the navigation arrows that disappear from limel-table.

https://user-images.githubusercontent.com/154725/212094251-ad927775-3994-409a-a744-78d75c70269e.mov

I had missed the "missing" navigation arrows in the docs, but taking a closer look now, they are indeed white when the system is in dark mode, even when Kompendium is in forced light mode:

<img width="374" alt="image" src="https://user-images.githubusercontent.com/154725/212095387-ef9c4624-5a0c-426f-bd2c-9891ffad3466.png">

Reply:
Oh yes. We don't have a separate shadows file. Like `light-mode-shadows-only.css`, that's why if you are in dark mode, shadows will get super dark.

And also the white arrows is because your OS is in dark mode. Long story, but they are specifically made for table in dark mode. Because otherwise, in dark mode, the arrows would be not visible.

Reply:
I think we need to find _some kind_ of fix for the arrows, but we'll fix it after merging this. I'm thinking white arrows with a thin black border, and black arrows with a thin white border, would at least make sure the arrows are visible regardless of background colour.